maestro/99-misc/C999001-receive.yaml

```diff
@@ -5,7 +5,7 @@
 
 appId: ${MAESTRO_APP_ID}
 tags:
-- android
+- all
```

Comment:
if this applies to all platforms, then delete the `C999001i-receive.yaml` which was for ios